packages/harmony/src/assets/styles/main.css

```diff
@@ -0,0 +1,15 @@
+@import './reset.css';
+@import './fonts.css';
+@import './spacing.css';
+@import './colors.css';
+@import './tokens.css';
+@import './border-radius.css';
+@import './shadows.css';
+
+#root *,
+body * {
+  box-sizing: border-box;
+  font-family: var(--font-family) !important;
```

Comment:
cool! so this is to make sure that inherit works for typography? would it be bad to have the global resets for that in the typography css instead of here?

Reply:
i was thinking about here vs there. felt fine to have all the global styles in the main sheet, but i'm not married to the idea

Reply:
yeah im good either way, just wondering for discoverability/isolation what's best. we can keep as is and think about it